tests/test_poller.cpp

```diff
@@ -41,9 +41,9 @@
 
 SETUP_TEARDOWN_TESTCONTEXT
 
-fd_t get_fd (void *socket_)
+zmq_fd_t get_fd (void *socket_)
```

Comment:
why the renaming churn? it's just tests, there's no need to namespace internal types

Reply:
fixed